apps/trading/components/rewards-container/active-rewards.tsx

```diff
@@ -85,6 +83,22 @@ export const applyFilter = (
   ) {
     return false;
   }
+
+  if (
```

Comment:
Are we missing some of the other suspended market states?
And would it be easier to reverse this if statement if we only want to show colour for active?